# CASE 7: noise stats

Since we have noise estimates from the ALMA query, we can cross-check that of ADMIT with ALMA, and show the pollution of the the missing short spacings

In [1]:
from astroquery.admit import ADMIT
import pandas as pd

pd.set_option('display.max_rows', None)
pd.set_option('display.max_columns', None)
pd.set_option('display.width', None)
pd.set_option('display.max_colwidth',25)


a = ADMIT()
a.check()

Found  /home/teuben/ALMA/study7/query/admit.db
Checking db.... 0
71 71 71
Database version: 27-feb-2022. core.py version: 26-feb-2022
header    : 1 entries
alma      : 115 entries
win       : 115 entries
lines     : 33 entries
sources   : 765 entries


## Line

Let's pick one selected MOUS first, the 7m data for NGC3504.  This is where one of the spw's has no signal, the CO and CS line are in two other spw's,and the fourth spw has fewer channels.





In [2]:
p1 = a.query(nchan='>2',obs_id='uid://A001/X87a/X70a')

select * from alma inner join win on (win.a_id = alma.id)  WHERE win.nchan>=2.0 AND alma.obs_id='uid://A001/X87a/X70a' 



In [3]:
# indeed there are 4 spw's
# p1
print(p1)


   id                obs_id target_name        s_ra      s_dec   frequency  \
0  51  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  229.366824   
1  52  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  230.733216   
2  53  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  243.689992   
3  54  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  245.564814   

         t_min  cont_sensitivity_bandwidth  sensitivity_10kms  \
0  57704.45099                  0.213557             6.807179   
1  57704.45099                  0.213557             6.514320   
2  57704.45099                  0.213557             6.754302   
3  57704.45099                  0.213557             7.155698   

           project_abstract                 obs_title  \
0  We propose to establi...  Pattern Speed in the ...   
1  We propose to establi...  Pattern Speed in the ...   
2  We propose to establi...  Pattern Speed in the ...   
3  We propose to establi...  Pattern Speed in the ...  

alma says cont has sensitivity  0.214 mJy/beam, but this is really over 4 spw's, each 1.9 GHz

alma says line10 should have 6.8 mJy.km/s;  channels are 4.8 km/s

ADMIT measured RMS in the spw's (rms_w) each at about 5 mJy/beam in a 4.8 km/s channel

## Continuum

In [4]:
p2 = a.query(nchan=1,obs_id='uid://A001/X87a/X70a')

select * from alma inner join win on (win.a_id = alma.id)  WHERE win.nchan=1 AND alma.obs_id='uid://A001/X87a/X70a' 



In [5]:
# there are 4, but there should be 5, the aggregative spw_19_21-23_25 is missing?
# p2
print(p2)

    id                obs_id target_name        s_ra      s_dec   frequency  \
0  108  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  229.366824   
1  109  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  230.733216   
2  110  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  243.689992   
3  111  uid://A001/X87a/X70a     NGC3504  165.796629  27.973328  245.564814   

         t_min  cont_sensitivity_bandwidth  sensitivity_10kms  \
0  57704.45099                  0.213557             6.807179   
1  57704.45099                  0.213557             6.514320   
2  57704.45099                  0.213557             6.754302   
3  57704.45099                  0.213557             7.155698   

           project_abstract                 obs_title  \
0  We propose to establi...  Pattern Speed in the ...   
1  We propose to establi...  Pattern Speed in the ...   
2  We propose to establi...  Pattern Speed in the ...   
3  We propose to establi...  Pattern Speed in the 

In [6]:
#  rough guess of what the aggregate cont rms would be from the 4 spw's
p2['rms_w'].mean()/2


0.0001840875

alma claims cont sens is 0.213 mJy (but over 4 spw's)

admit measures about 0.37 per spw, which would imply about 0.18 for the aggregrate. A bit lower than what alma claimed.

Note that for this continuum record, the line sensitivity is still reported because they share the same MOUS.

## All data?

The obvious thing to try is do this for all data, and get a ratio of what ADMIT measured and what ALMA claims. This would be a good check on any remaining problems in the ADMIT results

to be continued...